### PR TITLE
Correct MaterialData for TreeSpecies Blocks

### PR DESCRIPTION
**Pull request by hibo98**
I work with Trees and Leaves and noticed that no MaterialData is available for LEAVES_2 and LOG_2.

This Pull Request modifies MaterialData for Tree and Leaves that LEAVES_2, LOG_2 is useable.

--EDIT--

The same Pull Request is opened at SpigotHub [Bukkit#110](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/pull-requests/110/overview)

### CODE REVIEW

File: src/main/java/org/bukkit/material/Tree.java [hunk 1]
```diff
@@ -98,6 +107,7 @@ public BlockFace getDirection() {
                 return BlockFace.SELF;
         }
     }
+
```
Comment (mastercoms) [inline src/main/java/org/bukkit/material/Tree.java]: Thank you for this.

**Comment by mastercoms** [comment]
The code looks great, thanks! I'll test it and get back to you if I find any problems. If not, I'll merge it.

**Comment by hibo98** [comment]
I want to add more

**Comment by mastercoms** [comment]
Why did you close it?  :(

**Comment by gdude2002** [comment]
^, No need to close the PR, just keep committing

**Comment by hibo98** [comment]
It was closed by GitHub automaticly

**Comment by gdude2002** [comment]
It says, "hibo98:tree-material-data was force-pushed and no longer has any new commits."

**Comment by hibo98** [comment]
I was adding the commits from today, probably my failure...
I am working, I'll push it later.

**Comment by mastercoms** [comment]
Ok, thank you for your work on this!

**Comment by hibo98** [comment]
I am finished and hopefully the Commit appears now.

**Comment by gdude2002** [comment]
Yup, I can see it. Good stuff.

**Comment by mastercoms** [comment]
Sorry for the delay, but thank you for your contribution!

**Pull request merged**